core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -645,6 +665,15 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
       r <- e.fold(self.map(Left(_)) provide _, that.map(Right(_)) provide _)
     } yield r
 
+  /**
+   * Returns a successful effect if the value is `Left`, or fails with the error `None`.
+   */
+  def left[B, C](implicit ev: A <:< Either[B, C]): ZIO[R, Option[E], B] =
+    self.foldM(
+      e => ZIO.fail(Some(e)),
+      a => ev(a).fold(ZIO.succeed(_), _ => ZIO.fail(None))
```

Comment:
Super minor but I think you don’t need the `(_)`

Reply:
Nice!